src/ol/interaction/Translate.js

```diff
@@ -337,20 +338,26 @@ class Translate extends PointerInteraction {
    * features.
    * @param {import("../pixel.js").Pixel} pixel Pixel coordinate to test for intersection.
    * @param {import("../Map.js").default} map Map to test the intersection on.
-   * @return {import("../Feature.js").default} Returns the feature found at the specified pixel
+   * @return {Feature} Returns the feature found at the specified pixel
    * coordinates.
    * @private
    */
   featuresAtPixel_(pixel, map) {
     return map.forEachFeatureAtPixel(
       pixel,
+      /**
+       * @param {import("../Feature.js").FeatureLike} feature Feature.
+       * @param {import("../layer/Layer.js").default} layer Layer.
+       * @return {Feature|undefined} Feature that should be handled if found.
+       */
```

Comment:
Is this comment block necessary?

Reply:
It helps with type-safety. Without it `feature` and `layer` are of type `any` and no type-checking can happen.

Reply:
Hm, actually the parameter types seem to be inferred automatically, so could be removed.

Reply:
I found that the types were `any` when we were calling `bind` on an anonymous function, but that they are inferred without the bind.